docs/src/main/asciidoc/_templates/template-concepts.adoc

```diff
@@ -6,10 +6,10 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 TODO: See Quarkus documentation reference for details
 - Title should have an implied "Understanding..." in front. 
-- ID should start with 'concept-'
+- ID should end with '-concepts' (using the filename works)
```

Comment:
```suggestion
- ID should end with '-concept' (using the filename works)
```

Reply:
Suggestion: Rename the concept template filename to `template-concept.adoc`, i.e with no 's' at the end. Thanks.